flink-table/flink-table-planner/src/test/scala/org/apache/flink/table/planner/runtime/stream/sql/OverAggregateITCase.scala

```diff
@@ -159,6 +159,66 @@ class OverAggregateITCase(mode: StateBackendMode) extends StreamingWithStateTest
     assertEquals(expected.sorted, sink.getAppendResults.sorted)
   }
 
+  @Test
+  def testDenseRankOnOver(): Unit = {
```

Comment:
Maybe you can add some test cases for testing plan, not only just for ITCases.

Reply:
Hi, @xuyangzhong . Thanks for your review and sorry for delay response. I didn't find the UT of AggFunctionFactory, so I borrowed the test for ROW_NUMBER. And I haven't waited for the community's reply,  I can add corresponding unit tests later if necessary.